src/main/steps/common/yourhearings/hearings/utils.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Case } from 'app/case/case';
+import dayjs from 'dayjs';
+
+import { Attendee, CaseType, HearingsList, PartyType, hearingDay } from '../../../../app/case/definition';
+import { AppRequest } from '../../../../app/controller/AppRequest';
+import { HEARING_METHOD } from '../../../../steps/constants';
+import { getCasePartyType } from '../../../../steps/prl-cases/dashboard/utils';
+
+import { cy, en } from './content';
+
+export function getHearingMethod(req: AppRequest, attendees: Attendee[]): string {
+  const partyType = getCasePartyType(req.session.userCase, req.session.user.id);
+  const caseType = req.session.userCase.caseTypeOfApplication;
+  let partyData;
+  let hearingMethod;
+  const lang = req.session.lang === 'cy' ? cy : en;
+  if (caseType === CaseType.C100) {
```

Comment:
resuse getPartyDetails method which is already written

Reply:
As we have discussed earlier getPartyDetails method can't be reused here